Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/Visuals/VisualUtils.cs

```diff
@@ -205,5 +210,27 @@ static internal Vector3 GetLinkPosition(int linkIndex, ref Vector3[] cornerPoint
             return Vector3.zero;
         }
 
+        /// <summary>
+        /// returns the flatten indices to the corresponding flattenAxis mode
+        /// </summary>
+        /// <param name="flattenAxis">flatten axis mode that should be converted to indices</param>
+        /// <returns>flattened indices</returns>
+        internal static int[] GetFlattenedIndices(FlattenModeType flattenAxis)
+        {
+            if (flattenAxis == FlattenModeType.FlattenX)
+            {
+                return new int[] { 0, 4, 2, 6 };
```

Comment:
Slight nit, but could these arrays be declared `static readonly` outside the method, so we aren't `new`ing an array each time it's called?